tests/pos/ReWrite7.hs

```diff
@@ -47,7 +47,7 @@ const' x _ = x
       -> f  : (a -> m b)
       -> {v : m b | v == mbind om x f } @-}
 mbind :: OurMonad m -> m a -> (a -> m b) -> m b
-mbind = bind
+mbind (OurMonad b _) = b
```

Comment:
Is this edit required?

Reply:
Unfortunately yes. It seems like without it GHC 9 finds this code ambiguous. The edit provides enough information, and it's semantically equivalent.

Reply:
So, GHC 9 complains, no Liquid Haskell? This makes more sense :)